lib/puppet/provider/package/pkgin.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(packages)
   end
 
   def self.instances
-    pkgin(:list).split("\n").map do |package|
+    pkgin("-p", :list).split("\n").map do |package|
```

Comment:
Puppet users of this pkgin provider cannot use it with pkgin < 0.7.0. So perhaps we should remove the explicit `-p` and rely on the implied behaviour on pre and post 0.7.0 versions, while using the new format in the spec tests. Of course with the adjusted regex on line 16.